### PR TITLE
[OSDOCS-3006] Fix non-groupified API references

### PR DESCRIPTION
**Pull request by yselkowitz**
Use of non-groupified non-core objects was deprecated in OCP 4.7.

https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/cicd/builds/advanced-build-operations.html#builds-chaining-builds_advanced-build-operations
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/cicd/builds/custom-builds-buildah.html#builds-use-custom-builder-image_custom-builds-buildah
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/cicd/builds/creating-build-inputs.html#builds-using-secrets-as-environment-variables_creating-build-inputs
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/cicd/builds/creating-build-inputs.html#builds-using-secrets_creating-build-inputs

https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-eventrouter.html#cluster-logging-eventrouter-deploy_cluster-logging-eventrouter

https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/applications/deployments/what-deployments-are.html#deployments-and-deploymentconfigs_what-deployments-are
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/applications/deployments/deployment-strategies.html#deployments-lifecycle-hooks_deployment-strategies

https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/planning-your-environment-according-to-object-maximums.html#how-to-plan-according-to-application-requirements_object-limits

https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/image-streams-manage.html#images-imagestream-configure_image-streams-managing
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/image-streams-manage.html#images-imagestream-mapping_image-streams-managing
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using-imagestreams-with-kube-resources.html#images-managing-images-enabling-imagestreams-kube_using-imagestreams-with-kube-resources
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using_images/images-other-jenkins.html#images-other-jenkins-customize-s2i_images-other-jenkins
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using_images/images-other-jenkins.html#images-other-jenkins-kubernetes-plugin_images-other-jenkins

https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-secrets.html#nodes-pods-secrets-about-examples_nodes-pods-secrets

https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/networking/routes/route-configuration.html#nw-enabling-hsts-per-route_route-configuration
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/networking/ingress-operator.html#nw-http2-haproxy_configuring-ingress
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/networking/routes/secured-routes.html#nw-ingress-creating-a-passthrough-route_secured-routes
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/networking/routes/secured-routes.html#nw-ingress-creating-a-reencrypt-route-with-a-custom-certificate_secured-routes
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/networking/routes/secured-routes.html#nw-ingress-creating-an-edge-route-with-a-custom-certificate_secured-routes
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/networking/routes/route-configuration.html#nw-path-based-routes_route-configuration
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration

https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using-templates.html#templates-exposing-object-fields_using-templates
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using-templates.html#templates-waiting-for-readiness_using-templates
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using-templates.html#templates-writing-description_using-templates
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using-templates.html#templates-writing-parameters_using-templates
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using-templates.html#templates-writing_using-templates

https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.html#virt-attaching-vm-secondary-network-cli_virt-attaching-multiple-networks
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/importing_vms/virt-importing-virtual-machine-images-datavolumes.html#virt-importing-vm-datavolume_virt-importing-virtual-machine-images-datavolumes
https://deploy-preview-39200--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/advanced_vm_management/virt-schedule-vms.html#virt-setting-policy-attributes_virt-schedule-vms

### CODE REVIEW

**Comment by jboxman-rh** [comment]
@yselkowitz, if we can track down the right people from QE to look at this, I can get it merged. Thanks.

@sttts, does this PR make sense? Thanks!

**Comment by yselkowitz** [comment]
No idea who the "right people from QE" are for this, given that it runs through much of the docs.

File: modules/builds-chaining-builds.adoc [hunk 1]
```diff
@@ -23,7 +23,7 @@ The first build takes the application source and produces an image containing a
 
 [source,yaml]
 ----
-apiVersion: v1
+apiVersion: build.openshift.io/v1
```
Comment (mrunalp) [inline modules/builds-chaining-builds.adoc:26]: @adambkaplan ptal for build/image changes

File: modules/cluster-logging-eventrouter-deploy.adoc [hunk 2]
```diff
@@ -35,15 +35,15 @@ objects:
       name: eventrouter
       namespace: ${NAMESPACE}
   - kind: ClusterRole <2>
-    apiVersion: v1
+    apiVersion: rbac.authorization.k8s.io/v1
```
Comment (mrunalp) [inline modules/cluster-logging-eventrouter-deploy.adoc:38]: @soltysh could you review api/apps?

File: modules/nw-enabling-hsts-per-route.adoc [hunk 3]
```diff
@@ -18,7 +18,7 @@ HTTP strict transport security (HSTS) is implemented in the HAProxy template and
 .Example route configured with an annotation
 [source,yaml]
 ----
-apiVersion: v1
+apiVersion: route.openshift.io/v1
```
Comment (mrunalp) [inline modules/nw-enabling-hsts-per-route.adoc:21]: @Miciah ptal

File: modules/virt-attaching-vm-secondary-network-cli.adoc [hunk 4]
```diff
@@ -19,7 +19,7 @@ This procedure uses a YAML file to demonstrate editing the configuration and app
 +
 [source,yaml]
 ----
-apiVersion: v1
+apiVersion: kubevirt.io/v1
```
Comment (mrunalp) [inline modules/virt-attaching-vm-secondary-network-cli.adoc:22]: @fabiand ptal

File: modules/builds-chaining-builds.adoc (hunk 1 above)
Comment (adambkaplan) [reply modules/builds-chaining-builds.adoc:26]: lgtm

**Review by soltysh — APPROVED** [review]
workloads related api - lgtm

File: modules/nw-enabling-hsts-per-route.adoc (hunk 3 above)
Comment (yselkowitz) [reply modules/nw-enabling-hsts-per-route.adoc:21]: /cc @Miciah

**Comment by yselkowitz** [comment]
not specifically multi-arch related but fyi @kelbrown20

**Comment by bobfuru** [comment]
LGTM - inquired in docs peer review channel about QE ack for this.

**Comment by bobfuru** [comment]
@wangke19 for API Server and @kasturinarra for Workloads API, could you PTAL for QE acks? Thank you!

**Comment by kasturinarra** [comment]
LGTM from workloads side, thanks !!

**Comment by yselkowitz** [comment]
Ping @wangke19 for QE for API Server related sections.

**Comment by bobfuru** [comment]
@yselkowitz This feels like it has had sufficient exposure, scrutiny, and review via QE and Engineering. If no more updates or revisions are suggested, I'll merge tomorrow (24-March, US ET).

**Pull request merged**